r2/r2/public/static/css/reddit.less

```diff
@@ -448,9 +448,13 @@ ul.flat-vert {text-align: left;}
 .drop-choices.tabdrop {margin-top: 2px;}
 .dropdown-title.tabdrop { display: none }
 
-.drop-choices .choice.hidden {
+.drop-choices .choice.hidden,
+.drop-choices .choice.modsort {
     display: none;
 }
+.modsorts-on .drop-choices .choice.modsort {
```

Comment:
I didn't use a `body:not(.modsorts-on) ...` block just because some versions of IE don't actually support it, and they are used widely enough to potentially be significant.